units/meson.build

```diff
@@ -561,6 +561,10 @@ units = [
           'conditions' : ['ENABLE_BOOTLOADER', 'HAVE_OPENSSL', 'HAVE_TPM2'],
           'symlinks' : ['sysinit.target.wants/'],
         },
+        {
+          'file' : 'systemd-tpm2-clear.service.in',
+          'conditions' : ['ENABLE_BOOTLOADER', 'HAVE_OPENSSL', 'HAVE_TPM2'],
```

Comment:
Why BOOTLOADER and OPENSSL are necessary?

Reply:
all the tpm2 units pretty much have the same conditioning here. HAVE_TPM2 should be obvious. HAVE_OPENSSL is used by a lot of tpm code to convert to PEM or do various operations with. And ENABLE_BOOTLOADER is a corollary of the ConditionSecurity=measured-uki thing discussed above: if the boot loader is compile-time disabled, then the condition can never possible hold, hence we can disable the whole thing right-away.